gemfiles/5.0.gemfile

```diff
@@ -0,0 +1,6 @@
+gem "activesupport", "~> 5.0"
+gem "activemodel", "~> 5.0"
+gem "activerecord", "~> 5.0"
+gem "sqlite3", "~> 1.3.6"
+gem "mysql2"
+gem "pg"
```

Comment:
```suggestion
gem "mysql2", ">= 0.3.18", "< 0.6.0"
gem "pg", ">= 0.18", "< 2.0"
```
per https://github.com/rails/rails/blob/5-0-stable/activerecord/lib/active_record/connection_adapters/mysql2_adapter.rb#L4 and https://github.com/rails/rails/blob/5-0-stable/activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb#L2